dxr/plugins/python/tests/__init__.py

```diff
@@ -0,0 +1,18 @@
+from dxr.testing import SingleFileTestCase
+
+
+class PythonSingleFileTestCase(SingleFileTestCase):
+    """Test case suited to testing the Python plugin."""
+    source_filename = 'main.py'
+
+    @classmethod
+    def get_config(cls, config_dir_path):
+        config = super(PythonSingleFileTestCase, cls).get_config(config_dir_path)
```

Comment:
Here you need to say PythonSingleFileTestCase instead of cls. It's akin to what you do in an instance method's super call. Yeah, I know it sucks. But otherwise, this won't work correctly if somebody subclasses PythonSingleFileTestCase.